omero/sysadmins/unix/server-installation.txt

```diff
@@ -92,7 +93,7 @@ The following packages are required:
 +--------------+-------------+---------+
 | Python       | 2.7         | 2.6     |
 +--------------+-------------+---------+
-| PostgreSQL   | 9.4         | 9.3     |
+| PostgreSQL   | 9.4         | 9.2     |
```

Comment:
Although now I look, we still have 9.2 as the minimum elsewhere on this page and on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/server-postgresql.html so one page or the other will need correcting so we are consistent.